Wabbajack.App.Wpf/View Models/Gallery/ModListGalleryVM.cs

```diff
@@ -232,11 +232,12 @@ private async Task LoadModLists()
             try
             {
                 var modLists = await _wjClient.LoadLists();
+                var modlistSummaries = await _wjClient.GetListStatuses();
                 _modLists.Edit(e =>
                 {
                     e.Clear();
                     e.AddOrUpdate(modLists.Select(m =>
-                        new ModListMetadataVM(_logger, this, m, _maintainer, _wjClient, _cancellationToken)));
+                        new ModListMetadataVM(_logger, this, m, _maintainer, modlistSummaries, _wjClient, _cancellationToken)));
```

Comment:
I could have also added the network call inside each ModListMetaDataVM but it seemed better to reduce the network traffic and instead add it as an additional parameter to the method.